htdocs/lib2/search/search.gpx.inc.php

```diff
@@ -433,13 +433,21 @@ function search_output()
         $thisline = mb_ereg_replace('{logs}', $logentries, $thisline);
 
         // attributes
+        $addAttributes = [-1];  // dummy
+        if ($r['type'] == 10) {   // Drive-in
+            $addAttributes[] = 24;  // near the car / park and grab
+        }
         $rsAttributes = sql_slave(
-            '
+            "
             SELECT `gc_id`, `gc_inc`, `gc_name`
             FROM `caches_attributes`
             INNER JOIN `cache_attrib`
                 ON `cache_attrib`.`id`=`caches_attributes`.`attrib_id`
-            WHERE `caches_attributes`.`cache_id`=&1',
+            WHERE `caches_attributes`.`cache_id`='&1'
+            UNION
+            SELECT `gc_id`, `gc_inc`, `gc_name`
+            FROM `cache_attrib`
+            WHERE `id` IN (" . implode(',', $addAttributes) . ")",
             $r['cacheid']
```

Comment:
Yes, the double quotes are all intentional.
* lib1/lib2 escaped parameters are consistently quoted, as escaping can generally produce non-numbers.
  * Assuming that the supplied parameter #1 is an integer is locally unsafe and therefore deprecated.
* Quoting in lib1/lib2 SQL commands is consistently done with single quotes.
* Enclosing quotes of the statement here are consistently double quotes.